packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -92,7 +108,9 @@ export const useCustomValidity = (options: UseCustomValidityOptions) => {
       /**
        * Sync custom error with the native input validity.
        */
-      watchEffect(() => el.setCustomValidity(options.props.customError ?? ""));
+      watchEffect(() =>
+        el.setCustomValidity(getCustomErrors(options.props.customError)?.shortMessage ?? ""),
```

Comment:
Should we use both error messages in `setCustomValidity()` so no information is lost for screen readers?`

Reply:
like `${shortMessage}: ${longMessage}` ?

Reply:
done

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/fd02ae0b-e2e7-4f3c-b38d-d6c8f5e1e057)